client/stylesheets/_modal.scss

```diff
@@ -212,17 +212,17 @@
 }
 
 /* Radio Buttons */
-.modOfferRequestPaymentWrap ul li {
+.radioWrap ul li {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `radioWrap` should be written in lowercase with hyphens